CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [7.0.0]
+### Added
+- Add `getKnownPropertyNames` function ([#111](https://github.com/MetaMask/utils/pull/111))
+
+### Changed
+- **BREAKING:** Build the package as both CJS and ESM ([#115](https://github.com/MetaMask/utils/pull/115), [#124](https://github.com/MetaMask/utils/pull/124))
+  - It's no longer possible to import from the `dist` folder. Everything must be imported from `@metamask/utils`.
```

Comment:
Should we include the `semver` bump here since it's a security-based item?

Reply:
Maybe it's just the Snaps team that is inconsistent then... 😅